030_numberOfDigitsInteger.js

```diff
@@ -0,0 +1,7 @@
+// 30. Number of digits in an integer
+const numberOfDigits = (integer) => {
```

Comment:
Logic issue : Number should not read in as string. Number should be converted to integer and do not use string operations